docs/data-management/properties.md

```diff
@@ -6,6 +6,55 @@ sidebar_position: 2
 
 ## Entity properties
 
-### Assignment properties
+## Assignment properties
 
 ## Metric properties
+
+A class of properties that is sourced from Fact SQL.
+
+### Adding via Fact Definition
+
+![Configuring a Fact SQL](/img/properties/metric-property-fact-sql.png)
+
+Add a Metric property to any of your existing Facts or by creating a new one.
+
+This only needs to be configured once and re-use many times for any Metrics created.
+
+### Configure a Metric with a property filter
```

Comment:
Maybe add a description here:
"Metric Properties can be added as filters when creating Metrics. This will allow you to filter down to a particular value or values when calculating that metric. For example, you may want to create a metric called 'Purchases (USA)' to track purchases within a single country."